### DISCUSSION TITLE
The views We drawed suddenly disappear

### DISCUSSION THREAD

**Discussion by qinglin** [frangoteam/FUXA#1465] [Q&A]
Hi, Dear Team:
We met a problem yesterday, what we draw yesterday in the main view and data we bind suddenly disappear, the Fuxa server is still running but the data just gone away, is there any way to know what happen or is there any bug ? I checked the log, just said the  web api is not connected because the api server is down, no other information.

**Comment by MatthewReed303** [comment]
@qinglin this is a bug related to the undo, if you use undo to many times everything will disappear. A work around is to make a second view and leave it blank, if your contents disappear on your main view due to many undo, just click into the second view and back into the main view and it should all return again. @unocelli can we look into this issue? It's been there for quite some time now

**Reply by qinglin** [reply]
Hi, Matthew.
Thanks for your reply!! If we use the pro release version, would this problem be fixed, since it is kind of unsteady, we use about 3 hours to draw the mimic view, and suddenly it just gone away, we have to draw it again :(

**Reply by MatthewReed303** [reply]
@qinglin you can use save project as to create regular backups and also you can clone the view as backup before making any major changes. I don't think the Pro version will help in this situation.

**Reply by qinglin** [reply]
Thanks, we want to use the pro version of FUXA, and I have sent you the mail, is it possible to schedule a demo of the pro version of FUXA and gave us a basic introduction?
